app/realDealz/library.py

```diff
@@ -104,14 +104,20 @@ def m_get(self, url : str, p_headers : dict):
         _headers = {
             "Content-Type": "application/json"
         }
-        _headers.update(self.a_header)
         if p_headers is not None:
             _headers.update(p_headers)
-        result = json.loads(http.request(
-            'GET',
-            url,
-            headers=_headers
-        ).data)
+
```

Comment:
does this inherently change anything for the use of the library API i don't think so but i was just confused would this be a better method for get request?